tools/createTestReport/create_test_report.bat

```diff
@@ -34,7 +34,14 @@ rem Convert XML test report to TRLC.
 python test_result_xml2trlc.py ./%OUT_DIR%/%TEST_RESULT_REPORT_XML% ./%OUT_DIR%/%TEST_RESULT_REPORT_TRLC%
 
 rem Convert TRLC test report to Markdown.
-pyTRLCConverter --source=..\..\doc\sw-requirements --source=..\..\doc\sw-test --source=..\..\doc\models --exclude=..\..\doc\sw-requirements --exclude=..\..\doc\sw-test --source=%OUT_DIR%\%TEST_RESULT_REPORT_TRLC% -o=%OUT_DIR% --project=create_test_report.py --verbose markdown
+pyTRLCConverter --source=..\..\doc\sw-requirements --source=..\..\doc\sw-test --source=..\..\doc\models --exclude=..\..\doc\sw-requirements --exclude=..\..\doc\sw-test --source=%OUT_DIR%\%TEST_RESULT_REPORT_TRLC% -o=%OUT_DIR% --project=create_test_report_in_markdown.py --verbose markdown
```

Comment:
You may consider reformatting these commands to multiple lines for better readability. 

The ^ char at the end of a line in .bat files can be used for it.